### ISSUE TITLE
Device Discovery is unreliable

### ISSUE THREAD

**Issue by amp64** [amp64/sonosbugtracker#1]
There are different manifestations of this problem:

- No system found at startup, ie no devices found _at all_
- App starts up but doesn't find every device

**Comment by furacao14** [comment]
It may be just my impression, but it seems these issues are occurring mostly on the wireless devices connecting thru SonosNet.

**Comment by amp64** [comment]
No, connection issues seem to be across the board, seem independent of network transport. Certainly on my network which is 100% WiFi.

**Comment by furacao14** [comment]
Are you seeing any improvements on this? For a few months now I haven't had issues with my set of Beam 2 + Symfonisk bookshelf Gen 1 (as surrounds). I still have some lagginess and weird stuff happening with my Play:1, specially when grouped to the Beam. Perhaps, older devices don't have enough processing power/memory/etc required by the current firmware/device discovery process?
